### PR TITLE
Add note about Wake On Lan

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

**Comment by supersaiyanmode** [comment]
What do you think about wording something like this:

This library helps establish connection with the TV and control various aspects exposed over thhe WebSocket API (once it discovers the IP address). Not depending on other libraries (such as WOL) prevents client projects from having to pull in this transitive dependency (which might not be always needed). Further, waking up a sleeping TV requires us sending a magic packet outside of the WebSocket layer that this library tries to limit itself to. We would highly encourage you to perhaps use the discovered IP address, translate that to the MAC address (there are tons of libraries that do this for you) and then send the magic WOL packet to the TV (again, tons of libraries that do this).

**Comment by supersaiyanmode** [comment]
BTW, thank you so much, @Gustry. You are spending so much more time in responding to other PRs (I am unable to because of some personal reasons), I'll add you as a collaborator so you don't have to necessarily depend on me :)

File: README.md [hunk 1]
```diff
@@ -276,6 +276,11 @@ source_control.set_source(sources[0])      # .set_source(..) accepts an InputSou
 
 More controls coming soon!
 
+## Turn on the TV
+
+Turning on the TV is not included in this library because this can be done with a Wake On Lan (WOL) command.
+Use the MAC address of the TV and enable the WOL in the TV settings.
```
Comment (Gustry) [inline README.md:282]:
```suggestion
This library helps to establish connection with the TV and control various aspects exposed over the WebSocket API (once it discovers the IP address).

Not depending on other libraries, such as Wake On Lan (WOL) prevents client projects from having to pull in this transitive dependency (which might not be always needed). Further, waking up a sleeping TV requires us sending a magic packet outside of the WebSocket layer that this library tries to limit itself to.

We would highly encourage you to perhaps use the discovered IP address, translate that to the MAC address (there are tons of libraries that do this for you, or check behind the TV on a sticker or check on your router) and then send the magic WOL packet to the TV (again, tons of libraries that do this). The WOL must be enabled on the TV.
```

**Comment by Gustry** [comment]
Thanks for the invite. I got it.

I will squash this PR if it's fine like this. I put your suggestion.

**Pull request merged**